pkg/rpm/preupgrade

```diff
@@ -222,7 +225,7 @@ def calc_tpool_stats(stats):
     1073741824.0
     """
     data = {}
-    stats = [_f for _f in stats.split(' ') if _f]
+    stats = [_f for _f in stats.split(b' ') if _f]
```

Comment:
Is the `stats` parameter always going to be bytes?  If it's not, this will raise a TypeError.